texastribune/donations
# Portal CSS

#### What's this PR do?
Styles the UMP pages à la the design system and some local styles that mirror the directory structure of the DS.

From a JS perspective, everything is still hard-coded. This isn't linked to an API yet.

#### Why are we doing this? How does it help us?
So everything looks pretty.

#### How should this be manually tested?
+ `make`
+ `make restart`
+ Go to `/user-portal` and sign in with a test Auth0 non-staff account.
+ Click around the pages linked from the side nav. Make sure everything looks pretty and not broken and in line with these [mockups](https://3.basecamp.com/3098728/buckets/10357222/vaults/1746618729).

#### How should this change be communicated to end users?
NA right now.

#### Are there any smells or added technical debt to note?
@ashley-hebler Please let me know if there are places I'm not using our shared classes properly or am duplicating stuff that's already accounted for in our utilities. I've also got a running list of stuff I think we should eventually add to the DS based on my experience styling this app.

As far as repo-specific styles go, I created `static/sass/ds`, which contains a mirror of the DS' directory structure. That's where my UMP styles are. I know the top level `static/sass` also mirrors that structure, but those partials of course predate the DS and thus I wanted to explicitly distinguish between pre- and post-DS worlds.

#### What are the relevant tickets?
https://3.basecamp.com/3098728/buckets/10357222/todos/1720708167

#### Have you done the following, if applicable:
* [ ] Added automated tests? *(later)*
* [ ] Tested manually on mobile? *(browser testing will occur later)*
* [ ] Checked for security implications? *(later)*
* [ ] Updated the documentation/wiki? *(later)*
